requirements.txt

```diff
@@ -1,4 +1,6 @@
-lupa==1.3
```

Comment:
@OHerasym, please, update readme file too. You replaced lupa with lunatic python, this should be written somehow.